ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -725,6 +725,19 @@ tests:
     - ref: idp-htpasswd
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f4-disasterrecovery
+  cron: 7 1 * * 2,6
+  steps:
+    cluster_profile: vsphere-2
```

Comment:
Since https://github.com/openshift/release/pull/52585, `vsphere-2` get retired, I think you need to rebase your branch to update it to a correct one.

Reply:
updated.